packages/salesforcedx-vscode-soql/package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
   "soqlCustom_soql": "SOQL Builder",
-  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder"
-}
\ No newline at end of file
+  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder",
+  "soql_builder_toggle": "Toggle SOQL Builder"
```

Comment:
@ekapner - This is a screen shot of where this shows when the user hovers over the icon for a moment.

<img width="973" alt="Screen Shot 2020-12-20 at 6 07 01 PM" src="https://user-images.githubusercontent.com/599418/102729623-ab636680-42ee-11eb-8ec0-00aac3e9b068.png">

Reply:
@dehru - can we say "Switch to SOQL Builder" or "Open in SOQL Builder" ?